File: python/283.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        num_z = 0

        for i in range(len(nums)):
            if(nums[i]==0):
                num_z+=1
            else:
                nums[i], nums[i-num_z] = nums[i-num_z], nums[i]

File: cpp/283.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int num_z = 0;
        for(int i=0;i<nums.size(); i++) {
            if(nums[i]==0)
                num_z++;
            else
                swap(nums[i-num_z],nums[i]);
        }
    }
};